agentassist/src/app/pages/dashboard/customer-aspect/customer-aspect.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, EventEmitter, Input, OnInit, Output, SimpleChanges } from '@angular/core';
+import { IDashboardFilter } from '../dashboard-filters/dateFilter.model';
+import { actualvsDisplayTitle, DASHBORADCOMPONENTTYPE, VIEWTYPE } from '../dashboard.cnst';
+import { DashboardService } from '../dashboard.service';
+import { ServiceInvokerService } from '@kore.services/service-invoker.service';
+import { AuthService } from '@kore.services/auth.service';
+
+@Component({
+  selector: 'app-customer-aspect',
+  templateUrl: './customer-aspect.component.html',
+  styleUrls: ['./customer-aspect.component.scss']
+})
+export class CustomerAspectComponent implements OnInit {
+
+  @Input() viewType : string;
+  @Input() filters : IDashboardFilter;
+  @Input() customerDropdownSelection : string;
+  @Input() widgetData : any;
+
+  @Output() openSliderChild = new EventEmitter();
+
+  public customerAspectTabvsImage  = actualvsDisplayTitle.CUSTOMER_ASPECT_TYPE_VS_IMAGE;
+  customerAspectTabsList : any = actualvsDisplayTitle.CUSTOMER_ASPECT_TABS_LIST;
+  public DASHBORADCOMPONENTTYPELIST = DASHBORADCOMPONENTTYPE;
+  public VIEWTYPELIST = VIEWTYPE;
+
+  customerTabSelection : string = "All";
+  customerAspectData : any = [];
+  customerAspectAcutalData : any;
+  onChangeCall: boolean = false;
+  botId:any;
+  params = {
+    streamId: ''
+  }
+  payload: any = {
+    "startDate": "",
+    "endDate":"",
+    "sessionType" : "all", // agent or customer or all
+    "dataType" : "all", // all or faqs or articles or automations
+    "experience" : "",  // chat or voice
+    "skip":0,  // pages to skip (default 0)
+    "limit":3, // number of record to be fetched
+    "fetched":0  // count of previously fetched responses (default 0)
+
+  }
+
+  constructor(private dashboardService : DashboardService, private service : ServiceInvokerService, private authService : AuthService) { }
+
+  ngOnInit(): void {
+    this.params.streamId = this.authService.smartAssistBots[0]._id;
```

Comment:
@SandeepBa-Kore we are calling bot id with different method no? why we have changed here?